### DISCUSSION TITLE
Editable Fonts

### DISCUSSION THREAD

**Discussion by Heparinsky** [spacecowboy/Feeder#196] [Ideas]
Hello Dear Jonas Kalderstam!
I appreciate your donation to the community and also thank you for your great app.
Is it on your roadmap to add an option in order to style feeds with favorite fonts? I am using your app to read Farsi articles. Most devices don't have a readable and catching font for this language and it becomes annoying after some time... It would be great if the app had such an option.

Thanks again for your patience and response.

**Comment by spacecowboy** [comment]
It's one of the "would ike to have" things, but since it's not something I am likely to use myself it is unlikely that I will spend a lot of time on it.

**Reply by Heparinsky** [reply]
Understandable...
Thanks for your response🙏🏻
